### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by alexjhawk**
Updated the issue templates to resolve an issue
where the only option for the 'Do I have a custom
or modified version of the application?' question
was 'Yes'.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/05-other-report.yml [hunk 1]
```diff
@@ -15,13 +15,15 @@ body:
     attributes:
       value: |
         Please provide a clear and concise description of the report you are submitting.
-  - type: checkboxes
+  - type: dropdown
     attributes:
       label: 'Do I have a custom or modified version of the application?'
-      description: 'Some questions or concerns may be specific to custom or modified versions of the application.'
+      description: 'Some bugs may be specific to custom or modified versions of the application.'
```
Comment (TomKimsey) [inline .github/ISSUE_TEMPLATE/05-other-report.yml:21]: This seems unrelated to the commit message

Comment (TomKimsey) [reply .github/ISSUE_TEMPLATE/05-other-report.yml:21]: And actually it looks like a copy/paste mistake

Comment (alexjhawk) [reply .github/ISSUE_TEMPLATE/05-other-report.yml:21]: It was

**Review by TomKimsey — CHANGES_REQUESTED** [review]
resolve copy/paste mistake

**Comment by alexjhawk** [comment]
> resolve copy/paste mistake

Resolved

**Review by TomKimsey — APPROVED** [review]

**Review by it-hms — APPROVED** [review]
